accio-server/src/main/java/io/accio/server/module/ConnectorConfigModule.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.accio.server.module;
+
+import com.google.inject.Binder;
+import io.accio.base.config.BigQueryConfig;
+import io.accio.base.config.PostgresConfig;
+import io.airlift.configuration.AbstractConfigurationAwareModule;
+
+import static io.airlift.configuration.ConfigBinder.configBinder;
+
+public class ConnectorConfigModule
```

Comment:
Why we need to pull up the config to this module? I believe they should be close who use. And some config do not need to bind when the data source type is not with them.

Reply:
When ConfigManager be created, all of them should be injected.

Reply:
Not make sense. If the scenario is data source type is PG, why we should assert BQ config be injected?

Reply:
For the current architecture, we load all config when server start but switch them in runtime.